osbs/build/build_request.py

```diff
@@ -603,6 +604,33 @@ def render_koji_promote(self, use_auth=None):
                         "specified")
             self.dj.remove_plugin("exit_plugins", "koji_promote")
 
+    def render_koji_tag_build(self):
+        phase = 'exit_plugins'
+        plugin = 'koji_tag_build'
+        if not self.dj.dock_json_has_plugin_conf(phase, plugin):
+            return
+
+        if not self.spec.kojihub.value:
+            logger.info('Removing %s because no kojihub was specified', plugin)
+            self.dj.remove_plugin(phase, plugin)
+            return
+
+        if not self.spec.koji_target.value:
+            logger.info('Removing %s because no koji_target was specified', plugin)
+            self.dj.remove_plugin(phase, plugin)
+            return
+
+        self.dj.dock_json_set_arg(phase, plugin, 'kojihub', self.spec.kojihub.value)
+        self.dj.dock_json_set_arg(phase, plugin, 'target', self.spec.koji_target.value)
+
+        if (self.spec.koji_use_kerberos.value and
+                self.spec.koji_kerberos_principal.value and
+                self.spec.koji_kerberos_keytab.value):
+            self.dj.dock_json_set_arg(phase, plugin, 'koji_principal',
+                                      self.spec.koji_kerberos_principal.value)
+            self.dj.dock_json_set_arg(phase, plugin, 'koji_keytab',
+                                      self.spec.koji_kerberos_keytab.value)
```

Comment:
I think we need to come up with a way to better make kojihub available to plugins. Something like this https://github.com/projectatomic/atomic-reactor/issues/478.